crates/autopilot/src/arguments.rs

```diff
@@ -130,6 +130,9 @@ pub struct Arguments {
     /// in COW base units
     #[clap(long, env)]
     pub cip_14_reward_cap: Option<f64>,
+
+    #[clap(long, env, default_value = "0.97")]
```

Comment:
I think 3% spread might be a little too tight for long-tail tokens. I think something more like 10% may be more reasonable. Maybe @harisang can comment as they have used some heuristics for filtering out mis-prised orders in our optimization solvers in the past.

That being said, I would actually use a default that allows all orders through (so `0.0`) and then only manually configure this value in our deployment configuration. My rationale is that when we are doing local testing, I don't want to have to think about what price my limit order needs to be to exercise the quoting code. Don't feel strongly about this point though.

Reply:
I guess Marco and Tom may have done these nice preprocessing steps, but I am not really familiar with them.

However, 3% definitely sounds low. I would be very liberal with this number (e.g., 10-15%), although I expressed my concern about this altogether [here](https://github.com/cowprotocol/services/issues/740) as well, labeling it as order censorship in a way.